### ISSUE TITLE
Style Timetable Upload Page

### ISSUE THREAD

**Issue by jessicafenwick** [SE701-T1/frontend#20]
**Describe the task that needs to be done.**
Add styling to the Timetable upload page once it has been created. This issue scope covers only the styling of the component, no additional functionality is required.

**Describe how a solution to your proposed task might look like (and any alternatives considered).**
Timetable upload page is styled to follow the figma designs

![image](https://user-images.githubusercontent.com/62319771/159149666-2adce224-254a-4bb7-8f1e-1e3628bd4297.png)

**Notes**
Depend on #17

**Comment by ff-dev-45** [comment]
discussed in team meeting 3/20/22, left as future work
